boltdesignsystem/bolt
# Hotfix/bolt animate ie 11 fixes

## Jira

https://app.asana.com/0/1126340469288208/1142059318351865

## Summary

Fix  bolt-animate `triggerAnims` for IE 11,

## Details

Fix `triggerAnims` for IE 11, which failed because Set polyfill was absent on second load after build

## How to test

Go to with/without page on IE 11, see it working. Refresh, see it work again.
http://localhost:3000/pattern-lab/patterns/06-experiments-with-without-00-with-without/06-experiments-with-without-00-with-without.html
Test the isolated triggerAnims test on IE 11. Refresh, test again.
http://localhost:3000/pattern-lab/patterns/06-experiments-with-without-06-with-without-overlay-animation-test/06-experiments-with-without-06-with-without-overlay-animation-test.html
